src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -173,6 +173,33 @@ sequences using different contrast enhanced images. The label is the name of the
 contrast agent. The key `ContrastBolusIngredient` MAY be also be added in the
 JSON file, with the same label.
 
+This entity is used to indicate which component of the complex representation of the MRI signal is represented in voxel data.
+The [`part-<label>`](../99-appendices/09-entities.md#part) key/value pair is associated with the DICOM tag `0008,9208`.
+Allowed label values for this entity are `phase`, `mag`, `real` and `imag`, which are typically used in `mag/phase` or `real/imag` pairs.
+For example:
+
+```Text
+sub-01_part-mag_T1w.nii.gz
+sub-01_part-mag_T1w.json
+sub-01_part-phase_T1w.nii.gz
+sub-01_part-phase_T1w.json
+```
+
+Phase images MAY be in radians or in arbitrary units.
+The sidecar JSON file MUST include the units of the `phase` image.
+The possible options are `rad` or `a.u.`.
+For example:
+
+sub-01_part-phase_T1w.json
+
+```Text
+{
+   "Units": "rad"
+}
+```
+
+When there is only a magnitude image of a given type, the `part` key MAY be omitted.
+
```

Comment:
Having removed section headers, the flow of this is now strange. We start with "This entity" and don't reference `part` until the next sentence. The `ContrastBolusIngredient` seems like it should be associated with `ce-`, which is defined above `part-`.

I've proposed #615 for promoting `Units` beyond this one context, which I think would also have the side effect of allowing this section to be a bit more compact and improve the flow.

Reply:
I think the text around `ContrastBolusIngredient` should be changed so it doesn't sound like it's talking about anatomical MRI-specific metadata in general, but for this PR I've just moved that paragraph up to be back with the `ce` entity definition.